docs/index.txt

```diff
@@ -2,14 +2,14 @@
 Django documentation
 ====================
 
-.. rubric:: Everything you need to know about Django.
+.. rubric:: Everything about Django.
```

Comment:
This changes the meaning from "everything you need to know" to literally "everything" about Django. The documentation strives to tell people what they need to know. Some things are left undocumented intentionality because they are internal implementation details.

Reply:
oh thx....ill edit this one out and then make a pull request again...sorry